01week/datatypes.js

```diff
@@ -0,0 +1,62 @@
+"use strict";
+
+//Write a JavaScript program to display the current day and time.
+
+const todaysDate = new Date; //declare a constant variable called todaysDate. Use method new Date; to define the variable as the current date. 
+console.log(todaysDate); //print todaysDate.
+
+//Write a JavaScript program to convert a number to a string.
+
+const num = 1982; //declare a constant variable called num and define it as 1982.
+const numToString = num.toString(); //declare a constant variable called numToString. use method toString to convert num to a string.
+console.log(numToString); //print the new string
+
+//Write a JavaScript program to convert a string to the number.
+
+const myString = '9175729249';
+const myNewNum = Number(myString);
+console.log(myNewNum);
+
+const myContact = parseInt("9175729249 for Stiles");
+console.log(myContact);
+
+/*Write a JavaScript program that takes in different datatypes and prints out whether they are a:
+Boolean
+Null ****
+Undefined
+Number
+NaN *******
+String*/
+
+console.log(typeof false)
```

Comment:
This needs to be a function that takes in an argument and returns it's data type